lib/tasks/attendance_csv_importer.rake

```diff
@@ -4,7 +4,9 @@ task attendance_csv_importer: :environment do
   # Pull single file from Wonderschool URL
   # Process single file through Attendance Processor
   # Archive file to S3
-  AttendanceCsvImporter.new.call
+  start_date = ENV['wonderschool_attendance_import_start_date']&.to_date || 1.year.before
```

Comment:
oh shoot!  I forgot here:
Can you document this env var in the `.env.sample` file? And also our standard for env vars is to do them in all caps, can you change these two to all caps here as well?